### ISSUE TITLE
window is undefined when creating worker-in-worker

### ISSUE THREAD

**Issue by skratchdot** [webpack-contrib/worker-loader#13]
I created pull #12 to fix the issue I'm seeing.  Not sure if you want to pull it in or what.  All I did was wrap:

``` javascript
(function (w) {
// a copy of the existing code with "window" being replaced by "w"
}(this));
```

The issue popped up when I started creating inline workers inside of inline workers...

**Comment by skratchdot** [comment]
@sokra - Adding you to this thread so you see it :)

**Comment by skratchdot** [comment]
Actually, I closed #12 until I have more time to test.
